pepper-apis/dsm-core/pom.xml

```diff
@@ -239,6 +239,16 @@
             <artifactId>gson</artifactId>
             <!-- <version>2.8.5</version> -->
         </dependency>
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <version>5.2.0</version>
```

Comment:
versions should go into https://github.com/broadinstitute/ddp-study-server/blob/develop/pepper-apis/pom.xml